libbuteosyncfw/profile/ProfileManager.cpp

```diff
@@ -1254,7 +1258,7 @@ QDateTime ProfileManager::getNextRetryInterval(const SyncProfile* aProfile)
        !iSyncRetriesInfo[aProfile->name()].isEmpty())
     {
        quint32 mins = iSyncRetriesInfo[aProfile->name()].takeFirst();
-       nextRetryInterval = QDateTime::currentDateTime().addSecs(mins * 60); 
+       nextRetryInterval = QDateTime::currentDateTime().addSecs(mins * 60);
```

Comment:
Not related to this patch per se, but since this doesn't use UTC timestamps, I wonder if it's susceptible to problems if the device's timezone settings change.  I guess it would just be a missed-sync-trigger, so not a huge issue.